src/wd.rs

```diff
@@ -373,3 +372,440 @@ impl NewSessionResponse {
         }
     }
 }
+
+/// The builder of [`PrintConfiguration`].
```

Comment:
This is now quite a lot of code. Can we move the print stuff to a `mod print` and then re-export just the `pub` types here?